pkgs/tools/inputmethods/ibus/default.nix

```diff
@@ -29,12 +30,14 @@ stdenv.mkDerivation rec {
     "--disable-memconf"
     "--enable-ui"
     "--enable-python-library"
+    "--with-emoji-json-file=${nodePackages.emojione}/lib/node_modules/emojione/emoji.json"
```

Comment:
IIUC, it is a build-time dependency only. We should check it's not retained at run-time.